Documentation/ceph-object-multisite.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Object Multisite
+weight: 2250
+indent: true
+---
+
+# Object Multisite
+
+Multisite is a feature of Ceph that allows object stores to replicate its data over multiple Ceph clusters. 
+
+Multisite also allows object stores to be independent and isloated from other object stores in a cluster.
+
+When a ceph-object-store is created without the `multisite` section; a realm, zone group, and zone is created with the same name as the ceph-object-store.
+
+Since it is the only ceph-object-store in the realm, the data in the ceph-object-store remain independent and isolated from others on the same cluster.
+
+When a ceph-object-store is created with the `multisite` section, the ceph-object-store will join a custom created zone, zone group, and realm with a different than it's own.
+
+This allows the ceph-object-store to replacte it's data over multiple Ceph clusters.
+
+To review core multisite concepts please read the [ceph-multisite design overview](/design/ceph/object/ceph-multisite-overview.md).
+
+## Prerequisites
+
+This guide assumes a Rook cluster as explained in the [Quickstart](ceph-quickstart.md).
+
+# Creating Object Multisite
+
+If an admin wants to set up multisite on a Rook Ceph cluster, the admin should create:
```

Comment:
We'll need more background on how it requires two ceph clusters and a connection between them. But we can save that for the future PR when it's working end to end.